src/main/java/com/iota/iri/service/validation/impl/TransactionSolidifierImpl.java

```diff
@@ -290,11 +251,16 @@ private boolean isUnsolidWithoutEntryPoint(TransactionViewModel transaction, Has
      * {@inheritDoc}
      */
     @Override
-    public void addToBroadcastQueue(TransactionViewModel tvm){
-        if(transactionsToBroadcast.size() >= MAX_SIZE){
-            transactionsToBroadcast.remove();
+    public void addToBroadcastQueue(TransactionViewModel tvm) {
```

Comment:
Being lazy and not loading to my IDE, so apologies in advanced

Is this supposed to be `public` for sure, not `private` or just `@visibleForTesting`?

Reply:
It is meant to be public for the `TransactionValidator.quickSetSolid()` call, but looking at it again, the `SolidifyStage` takes care of placing it into broadcasting so i'll make it private again.